sphinx/builders/html/__init__.py

```diff
@@ -508,7 +509,8 @@ def prepare_writing(self, docnames: Set[str]) -> None:
             'script_files': self.script_files,
             'language': self.config.language,
             'css_files': self.css_files,
-            'sphinx_version': __display_version__,
+            'sphinx_version': sphinx_version,
```

Comment:
Shouldn't the `sphinx_version` still have `__display_version__`? I see the need to have `'sphinx_version_tuple': sphinx_version`, but I don't understand why do we need to change the `sphinx_version` dictionary key.

Reply:
Yes you are right originally I changed the existing variable to be a tuple but I reverted that and added a new variable instead. I must have forgotten to revert this part of the code. I will fix that and update the patch.